.azure/pipelines/ci.yml

```diff
@@ -234,14 +234,15 @@ stages:
                 $(WindowsSignLogArgs)
         displayName: Code sign packages
 
-      # Windows installers bundle both x86 and x64 assets
+      # Windows installers bundle x86/x64/arm64 assets
       - script: ./eng/build.cmd
                 -ci
                 -noBuildRepoTasks
                 -sign
                 -buildInstallers
                 -noBuildNative
                 /p:DotNetSignType=$(_SignType)
+                /p:BuildInParallel=false
```

Comment:
Could you look at recent builds of 'main' w/o this change and let us know whether this significantly slows down the job❔ This is often the longest job in official builds and we need to be careful about build perf regressions.

More generally, the problems you hit earlier indicate a wider problem w/ up-to-date (not fast up-to-date) checks in our native projects. The native project references shouldn't cause any rebuilds of the work done in earlier steps. Please file a fairly urgent issue to get to the root of this.

Reply:
Sure filed https://github.com/dotnet/aspnetcore/issues/40486

Reply:
The build installers step only takes 5 minutes in total, in a main build today it looks like it took 4:41, so not talking about a lot of time since there's only a handful of cpp files in the installers, https://dev.azure.com/dnceng/public/_build/results?buildId=1638321&view=logs&j=1b89928a-2219-5ef9-602f-f95beb3da4dc was the reference build

Looks like there might still be a race in here, since a second build looks like it failed after I just removed the artifacts from the yml

Reply:
@dougbu any ideas why the installers build ends up calling clean on the arm64 pdbs, looks like this is what causes the sporadic problem where the iiscommon.pdb is missing:  There doesn't appear to even be any special arm64 settings in the vcxproj: https://github.com/dotnet/aspnetcore/blob/haok/arm64/src/Installers/Windows/AspNetCoreModule-Setup/IIS-Setup/IIS-Common/lib/IISSetup.CommonLib.vcxproj

![image](https://user-images.githubusercontent.com/6537861/156423936-7ab80aa0-95a1-429c-8ecb-60c969389449.png)

Reply:
Adding `@(Clean)` items isn't the same thing as executing the `Clean` target. Do you see matches for `$target Clean` in your binary log❔

Reply:
Okay I think the issue is the arm64 references are not behaving like the existing x64/x86, I'm not sure why, I added them just like the other ones: in Directory.Build.props 

https://github.com/dotnet/aspnetcore/blob/a37bad27e69471999cf469e39efbdbc61f303077/src/Installers/Windows/AspNetCoreModule-Setup/Directory.Build.props#L74

Reply:
Guessing this is the result of letting everything != arm go through, I'm reverting that change I guess